probe-rs/src/architecture/riscv/communication_interface.rs

```diff
@@ -1140,6 +1143,11 @@ impl RiscvCommunicationInterface {
     pub fn read_csr_progbuf(&mut self, address: u16) -> Result<u32, RiscvError> {
         log::debug!("Reading CSR {:#04x}", address);
 
+        // csr addresses can only be 12bits
+        if address > 0xFFF {
```

Comment:
Can we make this a constant and the one below as well? :) And maybe add a reference to the docs? :)

Reply:
Added `RISCV_MAX_CSR_ADDR` with a link to the CSR spec.